.gitignore

```diff
@@ -11,4 +11,5 @@
 /bin/
 /composer.phar
 /.idea
-/.vagrant
\ No newline at end of file
+/.vagrant
+node_modules
```

Comment:
Not necessary, this excludes the folder and it's contents. No need to prepend with a slash. :)